# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/powerpc/yaboot/prefix.patch
# ROCK Linux is Copyright (C) 1998 - 2005 Clifford Wolf
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- yaboot-1.3.10/Makefile.orig	1904-01-01 00:59:57.000000000 +0000
+++ yaboot-1.3.10/Makefile	1904-01-01 01:07:44.000000000 +0000
@@ -7,7 +7,7 @@
 DEBUG = 0
 # make install vars
 ROOT =
-PREFIX = usr/local
+PREFIX = usr
 MANDIR = man
 # command used to get root (needed for tarball creation)
 GETROOT = fakeroot
